vaadin-element-mixin.html

```diff
@@ -34,6 +34,14 @@
           window.Vaadin.registrations.push(this);
         }
       }
+      ready() {
+        super.ready();
+        if (document.doctype === null) {
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/API/Document/doctype it should be possible to check the actual doctype:

`if (document.doctype.name !== 'html') {`

Reply:
I think this statement is enough for us:

> The property returns null if there is no DTD associated with the current document.
See https://developer.mozilla.org/en-US/docs/Web/HTML/Quirks_Mode_and_Standards_Mode

> In almost standards mode, there are only a very small number of quirks implemented.

Checked the "Transitional" DTD and it works:

```
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Reply:
Fine by me.

FYI, for "Transitional" DTD `document.doctype.name` is still `'html'`.